sbar.c

```diff
@@ -1261,12 +1263,19 @@ static void Sbar_DeathmatchOverlay(int start)
 	char num[12];
 	char myminutes[11];
 	char fragsstr[10];
+	char * ktxmode = Info_ValueForKey(cl.serverinfo, "mode");
 	player_info_t *s;
+	ti_player_t *ti_cl;
 	mpic_t *pic;
 	float scale = 1.0f;
 	float alpha = 1.0f;
+	float ca_alpha = 1.0f;	// alpha value for scoreboard elements during clan arena / wipeout
```

Comment:
As far as I can see, this initial value of 1.0f is never used. Right?
The variable gets a new value in line 1469, before we even use it. So there is no need to give an initial value.